Improve diff styling

TR4Android opened the pull request:
This pull request greatly improves diff readability by:
* adding fullline color highlighting (done with the new `DiffLineSpan`)
* adding line color option for diff info (the `@@` part)

Here are two screenshots comparing the changes:
*Before the changes:*
![screenshot_2016-01-21-22-51-52](https://cloud.githubusercontent.com/assets/10142068/12495812/1efdf3d8-c092-11e5-8975-7fc65655fbe0.png)
*After the changes:*
![screenshot_2016-01-21-22-49-07](https://cloud.githubusercontent.com/assets/10142068/12495820/2ae9c23a-c092-11e5-9b7a-33f1e9ba19fd.png)

alorma commented:
Looks really great!

May i ask something? Could you add a diff line text color?

I need it for a project and doesn't have much time.

I will accept this PR anyway if you don't, feel free to say no :)

Thanks

TR4Android commented:
@alorma diff line text color as in configuration of the text color when deleted, added etc.? I'll take a look a that.
You can still merge this PR, I'll send a new one once I've added text colors.

alorma commented:
Done

I will upload tomorrow a new version of the lib to maven

TR4Android commented:
@alorma No rush :smiley:

alorma commented:
Version uplaoded

`compile 'com.github.alorma:diff-textview:1.2.0'`
